BACKPORT.md

```diff
@@ -0,0 +1,39 @@
+- [Managing Backports in OpenSearch Plugins](#managing-backports-in-opensearch-plugins)
+  - [Features](#features)
+  - [Integration](#integration)
+  - [Usage](#usage)
+
+## Managing Backports in OpenSearch Plugins
+
+Backporting a PR/commit to a release branch is a manual process and can lead to missed backports before a release. The auto backport Github Action allows to create automatic backport PRs just by labelling them.
+
+### Features
+
+When the auto backport workflow is integrated with a repo, the following features are available:
+
+1. Allows auto backports on PRs that are merged and labelled (can be in any order). To backport a PR to 1.x, please add a label called `backport 1.x` to the PR and the backport workflow will do the rest.
+2. The backport workflow is now created by a Github App called `opensearch-trigger-bot` instead of `github-actions` which allows CI to run on such PRs.
+3. The backport branches are named in the form `backport/backport-<original PR number>-to-<base>`. These branches will be cleaned up by an auto delete workflow once the backport PR is merged.
+
+### Integration
```

Comment:
This seems odd to include this no? I would assume we would just want documentation on how to utilize the backporting mechanism with a link/reference to this equivalent doc in the opensearch plugins repository.

Reply:
Yeah I think this can be changed to a link in developer_guide or README file